frontend/src/pages/pipelines/global/modelCustomization/HyperparameterFields.tsx

```diff
@@ -0,0 +1,148 @@
+import { Grid, GridItem } from '@patternfly/react-core';
+import * as React from 'react';
+import HyperparameterEvaluationField from '~/concepts/pipelines/content/modelCustomizationForm/hyperparameterFields/HyperparameterEvaluationField';
+import HyperparameterLongNumberField from '~/concepts/pipelines/content/modelCustomizationForm/hyperparameterFields/HyperparameterLongNumberField';
+import HyperparameterNumericField from '~/concepts/pipelines/content/modelCustomizationForm/hyperparameterFields/HyperparameterNumericField';
+import { HyperparametersFormData } from '~/concepts/pipelines/content/modelCustomizationForm/modelCustomizationFormSchema/validationUtils';
+import { InputDefinitionParameterType } from '~/concepts/pipelines/kfTypes';
+import {
+  HYPERPARAMETER_DISPLAY_NAMES,
+  HYPERPARAMETER_LIST,
+  HyperparameterFields,
+} from '~/pages/pipelines/global/modelCustomization/const';
+
+type HyperparameterFieldsDisplayProps = {
+  hyperparameters: HyperparametersFormData;
+  isEmpty: boolean;
+  onChange: (hyperparameter: HyperparameterFields, hyperparameterValue: string | number) => void;
+};
+
+const HyperparameterFieldsDisplay: React.FC<HyperparameterFieldsDisplayProps> = ({
+  hyperparameters,
+  isEmpty,
+  onChange,
+}) => (
+  <Grid hasGutter>
+    {HYPERPARAMETER_LIST.slice(0, 8).map((field, index) => (
```

Comment:
you will need to support all fields even if not specified.

So the flow is
1. get all fields
2. subtract the ones used elsewhere
3. subtract a certain set of fields that UX explicitly wants to hide
4. render the rest -> with some being part of display name mapping and some just falling back to what the pipeline says